### DISCUSSION TITLE
锁定测试贴

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3667] [讨论]
到龙猫做完 `锁定` Label 时候可以试试有无效果（
现在使用 GitHub Mobile 进入到 Issue 或 Discussion，右上角三个点点开总有个锁定的选项，想试试又不敢点，怕搞砸了啥重要的东西……

如果 GitHub Moblie 点击 “锁定对话” 后，会这样：![Screenshot_20240410_195119.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/daafc21f-ae49-49c4-939a-9b60ff3b815a)
网页端则根本没有这个按钮……
不知道这算不算个 Bug……

**Comment by burningtnt** [comment]
GitHub Mobile 时常出现 UI 没做权限校验的问题…… 你看到这个警告是因为 GitHub 后端的权限校验起效了。

**Reply by allMagicNB** [reply]
咱也不知道有没有 API 可以调用查权限，也许人家压根就没做这种 API（？
不过我还不是 Collaborator 的时候，没有这个按钮，~~做了但没完全做？~~

**Reply by burningtnt** [reply]
你可以简单抓一下 GitHub 的网络通讯。就是 GitHub Mobile 写出问题了（

**Reply by allMagicNB** [reply]
感觉像是只会检查是否有特殊权限，有的话直接全部展示出来了（
我没抓包的环境，所以没法测试……

<details>

<summary>By the way</summary>

烧师傅你不是有自己的头像吗，为啥要用 GitHub 的初始头像（

</details>

**Comment by Deep-Dark-Forest** [comment]
呦，这不我#3535 吗

**Comment by wuliaodexiaoluo** [comment]
GitHub Mobile 做的就特别不好用，我安装的时候 APP 不给看代码，也不能修改，只能发布回复 Issue 和 Pull request

**Reply by allMagicNB** [reply]
你倒是更新啊……

**Reply by wuliaodexiaoluo** [reply]
我是想更新，可是 2FA 和 Passkey 是放在计算机上的，更新要重新登录，关键 Edge Mboile 不能同步 Passkey （事实上也不应该同步

**Reply by allMagicNB** [reply]
> 我是想更新，可是 2FA 和 Passkey 是放在计算机上的，更新要重新登录，关键 Edge Mboile 不能同步 Passkey （事实上也不应该同步

你就不能开个梯子给手机设 Passkey 吗？

**Reply by Ad-closeNN** [reply]
不用梯子也行...
[521github.com/settings/security](https://521github.com/settings/security)
还有GitHub Mobile加载不用梯子（又自带中文/未墙服务器/查看代码）

**Reply by allMagicNB** [reply]
> 不用梯子也行...
> [521github.com/settings/security](https://521github.com/settings/security)
> 还有GitHub Mobile加载不用梯子（又自带中文/未墙服务器/查看代码）

图片要

**Reply by Ad-closeNN** [reply]
哦哦这个确实

**Reply by wuliaodexiaoluo** [reply]
用镜像站不是最优解，在有代理的情况下

**Reply by wuliaodexiaoluo** [reply]
>>我是想更新，可是 2FA 和 Passkey 是放在计算机上的，更新要重新登录，关键 Edge Mboile 不能同步 Passkey （事实上也不应该同步
>
>你就不能开个梯子给手机设 Passkey 吗？

我有代理，但设置 Passkey 要登录，登录要 2FA，但是 Edge Mobile 的 Chromium 内核不支持插件（Edge 可以同步插件数据，2FA可以同步过来），我忘记存 Recover Code 了 （事实上是保存了我忘记放哪里）

**Reply by Deep-Dark-Forest** [reply]
> > 不用梯子也行...
> > [521github.com/settings/security](https://521github.com/settings/security)
> > 还有GitHub Mobile加载不用梯子（又自带中文/未墙服务器/查看代码）
> 
> 图片要

我这不要啊…开流量的情况下

**Reply by Deep-Dark-Forest** [reply]
> >>我是想更新，可是 2FA 和 Passkey 是放在计算机上的，更新要重新登录，关键 Edge Mboile 不能同步 Passkey （事实上也不应该同步
> >
> >你就不能开个梯子给手机设 Passkey 吗？
> 
> 我有代理，但设置 Passkey 要登录，登录要 2FA，但是 Edge Mobile 的 Chromium 内核不支持插件（Edge 可以同步插件数据，2FA可以同步过来），我忘记存 Recover Code 了 （事实上是保存了我忘记放哪里）

你可以通过https://github.com/settings/auth/recovery-codes 重新查看……

**Reply by WorldHim** [reply]
> 我是想更新，可是 2FA 和 Passkey 是放在计算机上的，更新要重新登录，关键 Edge Mboile 不能同步 Passkey （事实上也不应该同步

虽然但是更新为什么要重新登录

**Reply by wuliaodexiaoluo** [reply]
> > 我是想更新，可是 2FA 和 Passkey 是放在计算机上的，更新要重新登录，关键 Edge Mboile 不能同步 Passkey （事实上也不应该同步
> 
> 虽然但是更新为什么要重新登录

我也不道，但我更新完登录就没了（可能是刚好过期？

**Reply by wuliaodexiaoluo** [reply]
> > 不用梯子也行...
> > [521github.com/settings/security](https://521github.com/settings/security)
> > 还有GitHub Mobile加载不用梯子（又自带中文/未墙服务器/查看代码）
> 
> 图片要

我这不用
